roles/analytics-worker/tasks/custom.yml

```diff
@@ -15,3 +15,7 @@
 #----------------------------------------------------------------------------
 
 # Add configs for the custom files that are needed to be copied/replaced here.
+ - name: "Copy jar to bundle script"
+   template:
+     src: "{{ product_package_location }}/misc/jartobundle.sh"
+     dest: "{{ carbon_home }}/wso2/analytics/bin"
```

Comment:
Only include examples in the custom scripts.

Reply:
fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243